run-tests.js

```diff
@@ -215,22 +225,22 @@ const testExchange = async (exchange) => {
             { language: 'Python 3',       key: '--python',       exec: ['python3',   'python/ccxt/test/test_sync.py',  ...args] },
             { language: 'Python 3 Async', key: '--python-async', exec: ['python3',   'python/ccxt/test/test_async.py', ...args] },
             { language: 'PHP',            key: '--php',          exec: ['php', '-f', 'php/test/test_sync.php',         ...args] },
+            { language: 'PHP Async', key: '--php-async',    exec: ['php', '-f', 'php/test/test_async.php',   ...args] }
         ]
 
-        if (!skipSettings[exchange] || !!!skipSettings[exchange].skipPhpAsync) {
+        const allTests = allTestsWithoutTs.concat([
+            { language: 'TypeScript',     key: '--ts',           exec: ['node',  '--loader', 'ts-node/esm',  'ts/src/test/test.ts',           ...args] },
+        ]);
+
+        const specificLangSet = (Object.values (langKeys).filter (x => x)).length;
```

Comment:
this variable is a bit ambiguous and its purpose is not clear because it's a number but is treated like a bool `&& !specificLangSet`, i would rather make it a bool directly

Reply:
like this? (edited)

Reply:
Yeah it's better now!

Reply:
ok, now it also passes tests.